app/models/spree/option_value_decorator.rb

```diff
@@ -1,10 +1,11 @@
 Spree::OptionValue.class_eval do
 
   has_attached_file :image,
-    :styles        => { mini: '32x32>', normal: '128x128>' },
+    :styles        => { mini: '32x32>', small: '64x64>', normal: '128x128>' },
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.